src/OrchardCore.Modules/OrchardCore.Setup/Views/Setup/Index.cshtml

```diff
@@ -246,6 +245,22 @@
                 $pwd.focus();
             }
         });
+        $("#DatabaseProvider").change(function () {
+            var dataProvider = $("#DatabaseProvider option:selected").text();
+            var connectionString = "";
+            switch (dataProvider) {
+                case "Sql Server":
+                    connectionString = "Server=<em>localhost</em>;Database=<em>Orchard</em>;User Id=<em>username</em>;Password=<em>password</em>";
+                    break;
+                case "MySql":
+                    connectionString = "Server=<em>localhost</em>;Database=<em>Orchard</em>;Uid=<em>username</em>;Pwd=<em>password</em>";
+                    break;
+                case "Postgres":
+                    connectionString = "Server=<em>localhost</em>;Port=5432;Database=<em>Orchard</em>;User Id=<em>username</em>;Password=<em>password</em>";
+                    break;
+                default:
+            }
+            $("#connectionStringHint").html('@T["The connection string to your database instance. e.g., "]' + connectionString);
```

Comment:
In RTL isn't the string on the left?

In this case the best option is to create 3 hidden `<div>` that will contain the full translation for each database, and assign an id to it to load the text from javascript.